codebuild/bin/fuzz_corpus_download.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+# Clean the environment before copying corpuses from the S3 bucket.
+# The LD variables may interfere with certificate validation when communicating with AWS S3.
+unset LD_PRELOAD
```

Comment:
Can we unset these in a single place instead of having to repeat this for each script? Ideally `runFuzzTest.sh` would do this/clean up after itself.

Reply:
Since LD_PRELOAD was only active in the scope of runFuzzTest.sh, I was able to just remove this.